examples/carousel/css/carousel-2-tablist.css

```diff
@@ -0,0 +1,234 @@
+
+/* .carousel */
+
+.carouselTablist .carousel-inner {
```

Comment:
Can we make the class names consistent?

`.carousel-tablist` instead of `.carouselTablist`